ogr/services/github.py

```diff
@@ -358,8 +358,17 @@ def add_issue_labels(self, issue_id, labels) -> None:
 
     def get_pr_list(self, status: PRStatus = PRStatus.open) -> List[PullRequest]:
         prs = self.github_repo.get_pulls(
-            state=status.name, sort="updated", direction="desc"
+            # Github API has no status 'merged', just 'closed'/'opened'/'all'
+            state=status.name if status != PRStatus.merged else "closed",
+            sort="updated",
+            direction="desc",
         )
+
+        if status == PRStatus.merged:
+            prs = list(prs)  # Github PaginatedList into list()
+            for pr in prs:
+                if not pr.is_merged():  # parse just merged PRs
+                    prs.remove(pr)
         try:
             return [self._pr_from_github_object(pr) for pr in prs]
```

Comment:
I think we also need to change the `_pr_from_github_object` method to translate the status correctly. Or am I wrong?

e.g.:
```python
get_pr_list(status=PRStatus.all)
```

Here we get the `closed` state for all `merged`.

Reply:
Yes, I was thinking about what is better, the decision is on us. I'll change it.

Reply:
Your code is correct to this point, we only need to extend `_pr_from_github_object` to differentiate the closed and merged status.